packages/ui/src/contexts/shortcuts.tsx

```diff
@@ -0,0 +1,51 @@
+import {ComponentChildren, createContext} from 'preact';
+import {useContext, useState} from 'preact/hooks';
+
+export type Shortcut = {
+  key: string;
+  action: string;
+  isGlobal?: boolean;
+};
+
+export type ShortcutModules = 'global' | 'timeline' | 'none';
+type ShortcutsByModule = Record<ShortcutModules, Shortcut[]>;
+type ShortcutsState = {
+  currentModule: ShortcutModules;
+  shortcuts: ShortcutsByModule;
+  setCurrentModule?: (module: ShortcutModules) => void;
+};
+
+const initialshortcuts: ShortcutsByModule = {
+  global: [
+    {key: 'Space', action: 'Toggle playback', isGlobal: true},
+    {key: '←', action: 'Previous frame', isGlobal: true},
```

Comment:
We should use ligatures since the editor font supports them:
```suggestion
    {key: '<-', action: 'Previous frame', isGlobal: true},
```
This will make sure that the arrow width matches the rest of the font.